docs/ci.md

```diff
@@ -0,0 +1,248 @@
+# CI Tests
+
+For CI, OVN-Kubernetes runs the
+[Kubernetes E2E tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md)
+and some locally defined tests. 
+[GitHub Actions](https://help.github.com/en/actions)
+are used to run a subset of the Kubernetes E2E tests on each pull request. The
+local workflow that controls the test run is located in
+[ovn-kubernetes/.github/workflows/test.yml](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml).
+
+The following tasks are performed:
+- Builds OVN-Kubernetes
+- Checks out the Kubernetes source tree and compiles some dependencies
+- Installs KIND
+- Runs a matrix of End-To-End Tests using KIND
+
+Below describes how to update the set of tests that run and how to run these
+tests locally.
+
+## Updating CI Test Suite
+
+The tests are broken into a set of shards, which is just a grouping of tests,
+and each shard is run in a separate job in parallel. Below is an example of
+the shards (which may change in the future):
+- shard-n
+  - All E2E tests that match `[sig-network] N` and do NOT have P as their sixth
+  letter after the N (i.e. Roughly all `[sig-network] Networking ...` tests.)
+- shard-np
+  - All E2E tests that match `[sig-network] N` and DO have a P as their sixth
+  letter after the N. (i.e. Roughly all `[sig-network] NetworkPolicy ...`
+  tests.)
+- shard-s
+  - All E2E tests that match `[sig-network] S`. (i.e. Roughly all
+  `[sig-network] Services ...` tests.)
+- shard-other
+  - All remaining E2E tests that didn't match above.
+- control-plane
+  - All locally defined tests.
+
+Each of these shards is run in a HA setup and a non-HA setup. The regex
+expression for determining which E2E test is run in which shard, as well as the
+list of skipped tests is defined in
+[ovn-kubernetes/test/scripts/e2e-kind.sh](https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-kind.sh).
+
+The local tests are controlled in
+[test/scripts/e2e-kind.sh](https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-kind.sh)
+and the actual tests are defined in the directory
+[ovn-kubernetes/test/e2e/](https://github.com/ovn-org/ovn-kubernetes/tree/master/test/e2e).
+
+## Running CI Locally
+
+This section describes how to run CI tests on a local deployment. This may be
+useful for expanding the CI test coverage, or testing a private fix before
+creating a pull request.
+
+### Download and install kubetest
```

Comment:
sorry to make you change this again, but we just got rid of using kubetest:

https://github.com/ovn-org/ovn-kubernetes/pull/1246